rust/cardano-blockchain-types/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "cardano-blockchain-types"
+description = "Common Cardano Blockchain data types for use in both applications and crates"
+keywords = ["cardano", "catalyst",]
+version = "0.0.1"
+authors = [
+    "Steven Johnson <steven.johnson@iohk.io>"
+]
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[lints]
+workspace = true
+
+[dependencies]
+pallas = { version = "0.30.1", git = "https://github.com/input-output-hk/catalyst-pallas.git", rev = "9b5183c8b90b90fe2cc319d986e933e9518957b3" }
+pallas-crypto = { version = "0.30.1", git = "https://github.com/input-output-hk/catalyst-pallas.git", rev = "9b5183c8b90b90fe2cc319d986e933e9518957b3" }
+# pallas-hardano = { version = "0.30.1", git = "https://github.com/input-output-hk/catalyst-pallas.git", rev = "9b5183c8b90b90fe2cc319d986e933e9518957b3" }
```

Comment:
No, we may need it as we iterate adding features to this new crate.  For now I would rather it sit here, rather than have to hunt to find it in another crate.  We can remove it once this crate has settled down.